src/scripts/main.js

```diff
@@ -1,3 +1,47 @@
 'use strict';
 
-// write your code here
+const promise1 = new Promise((resolve, reject) => {
+  document.addEventListener('click', (e) => {
+    if (e.target.closest('.logo')) {
+      resolve('Promise was resolved!');
+    }
+  });
+});
```

Comment:
The promise `promise1` is set up to resolve when a click event occurs on an element with the class 'logo'. Ensure that the '.logo' class is present in your HTML to trigger this promise.